apps/meteor/app/livechat/server/lib/QueueManager.ts

```diff
@@ -213,6 +215,51 @@ export class QueueManager {
 		return Boolean(contact.channels.some((channel) => isVerifiedChannelInSource(channel, room.v._id, room.source)));
 	}
 
+	static async startConversation(
+		rid: string,
+		insertionRoom: InsertionModel<IOmnichannelRoom>,
+		guest: ILivechatVisitor,
+		roomInfo: IOmnichannelRoomInfo,
+		defaultAgent?: SelectedAgent,
+		message?: string,
+		extraData?: IOmnichannelRoomExtraData,
+		attempts = 3,
+	): Promise<{ room: IOmnichannelRoom; inquiry: ILivechatInquiryRecord }> {
+		const session = client.startSession();
+		try {
+			session.startTransaction();
+			const room = await createLivechatRoom(insertionRoom, session);
+			logger.debug(`Room for visitor ${guest._id} created with id ${room._id}`);
+			const inquiry = await createLivechatInquiry({
+				rid,
+				name: room.fname,
+				initialStatus: await this.getInquiryStatus({ room, agent: defaultAgent }),
+				guest,
+				message,
+				extraData: { ...extraData, source: roomInfo.source },
+				session,
+			});
+			const livechatSetting = await LivechatRooms.updateRoomCount(session);
+			if (livechatSetting) {
+				void notifyOnSettingChanged(livechatSetting);
+			}
```

Comment:
Send the notify only after the update has been commited.

As it is if the commit fails it'll send a new notification with every retry.

Reply:
I'm not sure but since `updateRoomCount` runs an `$inc`, this may also end up sending a notification with the wrong value if there are other places changing that value in parallel.

Reply:
Actually, i don't even know why we have this setting being incremented. Or propagated. Cannot see any place where it's being used and its a hidden setting.

Maybe we could use it for some metrics, but for sure doesn't need to be notified.

Reply:
Removed notification. Will create a task to check where this can be used... or to just remove it.